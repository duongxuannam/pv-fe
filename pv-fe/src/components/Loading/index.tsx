const Loading = () => {
  return (
    <div className="flex w-full max-w-md mx-auto overflow-hidden rounded-lg shadow-lg bg-gray-300 animate-pulse dark:bg-gray-800">
      <div className="h-60 w-max bg-gray-300 dark:bg-gray-600"></div>
    </div>
  )
}

export default Loading
