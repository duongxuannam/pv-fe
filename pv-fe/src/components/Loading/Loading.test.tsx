import { render } from "@testing-library/react"
import Loading from "."

test("loads and displays ui", async () => {
  // ARRANGE
  const view = render(<Loading />)

  // ACT
  // do something if need action

  // ASSERT
  expect(view).toMatchSnapshot()
})
