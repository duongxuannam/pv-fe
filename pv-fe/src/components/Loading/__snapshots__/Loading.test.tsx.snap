// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`loads and displays ui 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="flex w-full max-w-md mx-auto overflow-hidden rounded-lg shadow-lg bg-gray-300 animate-pulse dark:bg-gray-800"
      >
        <div
          class="h-60 w-max bg-gray-300 dark:bg-gray-600"
        />
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="flex w-full max-w-md mx-auto overflow-hidden rounded-lg shadow-lg bg-gray-300 animate-pulse dark:bg-gray-800"
    >
      <div
        class="h-60 w-max bg-gray-300 dark:bg-gray-600"
      />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
